py_ecc/bls/ciphersuites.py

```diff
@@ -106,15 +106,15 @@ def KeyGen(cls, IKM: bytes, key_info: bytes = b'') -> int:
     def KeyValidate(PK: BLSPubkey) -> bool:
         try:
             pubkey_point = pubkey_to_G1(PK)
-        except ValidationError:
+        except Exception:
             return False
 
         if is_inf(pubkey_point):
             return False
 
         try:
             pubkey_subgroup_check(pubkey_point)
-        except ValidationError:
+        except Exception:
```

Comment:
`Exception` is kind of broad, should we catch `ValidationError` and `ValueError`?